landing page/js/main.js

```diff
@@ -0,0 +1,74 @@
+
+function splittext() {
+	TweenMax.to('#main_info_cont', 1, 
+	{
+		opacity: 0,
+		delay: 2.5
+	}
+	)
+	;
```

Comment:
This kind of snippet is preferred `);`. 

Replace [Line 8 and 9](https://github.com/gawdsnitkkr/tedxnitkurukshetra-frontend/blob/da50cd9b4cd87d423144fb8c4828fa3da3ede4bc/landing%20page/js/main.js#L8-L9) with `);`.

Do the same at [L16](https://github.com/gawdsnitkkr/tedxnitkurukshetra-frontend/blob/da50cd9b4cd87d423144fb8c4828fa3da3ede4bc/landing%20page/js/main.js#L16), [L42](https://github.com/gawdsnitkkr/tedxnitkurukshetra-frontend/blob/da50cd9b4cd87d423144fb8c4828fa3da3ede4bc/landing%20page/js/main.js#L42), [L49](https://github.com/gawdsnitkkr/tedxnitkurukshetra-frontend/blob/da50cd9b4cd87d423144fb8c4828fa3da3ede4bc/landing%20page/js/main.js#L49).